README.md

```diff
@@ -255,6 +255,14 @@ juju config docker-registry \
 
 >Note: If any of the swift config options are set, they must all be set.
 
+It is possible to configure for domain & domainid. If both are specified then domain will be the default.
+To configure use keys:
+```
+storage-swift-domain=<val>
+or
+storage-swift-domainid=<val>
```

Comment:
`domain` and `domainid` are kinda like `tenant` and `tenantid`:

https://github.com/docker/docker.github.io/blob/master/registry/storage-drivers/swift.md

Either may be set.  I think supporting `foo` and `fooid` is redundant and increases the support burden for users that may not catch which one takes precedence. Let's drop `domainid` and just tell users about `domain`.